From 966d60c2dfb39f3377286c2670eafe058a74e634 Mon Sep 17 00:00:00 2001
From: Mao Minkai <maominkai@wxiat.com>
Date: Tue, 7 Jun 2022 17:41:32 +0800
Subject: [PATCH] sw64: vdso: change vdso version
Patch-mainline: NO, OTHERS
References: openEuler-22.03-LTS
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56P0Z

--------------------------------

Change vDSO version to LINUX_2.6 to improve compatibility with 3rd
party tools such as Linux Test Project(LTP).

Signed-off-by: Mao Minkai <maominkai@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Acked-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/kernel/vdso/vdso.lds.S | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/sw_64/kernel/vdso/vdso.lds.S b/arch/sw_64/kernel/vdso/vdso.lds.S
index 67a635d6dfaf..de1782ccb7b6 100644
--- a/arch/sw_64/kernel/vdso/vdso.lds.S
+++ b/arch/sw_64/kernel/vdso/vdso.lds.S
@@ -79,7 +79,7 @@ PHDRS
  */
 VERSION
 {
-	LINUX_2.6.39 {
+	LINUX_2.6 {
 	global:
 		__vdso_rt_sigreturn;
 		__vdso_gettimeofday;
-- 
2.34.1

